server/runtime/src/test/java/org/infinispan/server/configuration/ServerOverlayConfigurationTest.java

```diff
@@ -30,4 +31,60 @@ public void testOverlay() {
       GlobalConfiguration global = holder.getGlobalConfigurationBuilder().build();
       assertNotNull(global.module(ServerConfiguration.class));
    }
+
+   @Test
+   public void testOverlayTwice() {
+      Properties properties = new Properties();
+      properties.put(Server.INFINISPAN_SERVER_CONFIG_PATH, getConfigPath().toString());
+      Server server = new Server(getConfigPath().toFile(),
+              Arrays.asList(Paths.get("Base.xml"),
+              Paths.get("Overlay.yml"),
+              Paths.get("OverlayTwice.yml")),
```

Comment:
It would be nice if the file name was more descriptive of it's intent, e.g. "Overlay-ReplicatedCache.yml"